src/components/globalnavigasjon/GlobalNavigasjon.tsx

```diff
@@ -23,6 +23,7 @@ const StyledLi = styled.li`
 const StyledLink = styled(Link)`
   display: flex;
   justify-content: space-between;
+  text-decoration: none;
```

Comment:
Denne kan kanskje fjernes nå som det ligger i styles.less ?

Reply:
Siden målet er å ikke måtte trenge globale stiler som at alle `a` tags ikke skal ha `text-decoration`. Så tenker jeg egentlig at der vi ikke vil ha `text-decoration` bør være eksplisitt 😊
Ideelt sett burde vi kanskje brukt en standard aksel komponent for dette, men det får komme senere 😁